### PR TITLE
set up state file notification emails + welcome email being sent upon valid verification code

### PR DESCRIPTION
**Pull request by mrotondo**

### CODE REVIEW

File: app/controllers/mailgun_webhooks_controller.rb [hunk 1]
```diff
@@ -141,7 +141,8 @@ def update_outgoing_email_status
     email_to_update = (
       OutgoingEmail.find_by(message_id: message_id) ||
         VerificationEmail.find_by(mailgun_id: message_id) ||
-        OutgoingMessageStatus.find_by(message_id: message_id, message_type: :email)
+        OutgoingMessageStatus.find_by(message_id: message_id, message_type: :email) ||
+        StateFileNotificationEmail.find_by(message_id: message_id)
```
Comment (mpidcock) [inline app/controllers/mailgun_webhooks_controller.rb:145]: Very cool that this is already here

File: config/locales/en.yml [hunk 2]
```diff
@@ -1533,6 +1533,25 @@ en:
...
+            Hi,
+
+            We successfully created your FileYourStateTaxes account! We'll send you updates on your state tax return status once you submit it. While you are filing, your responses will be saved as you go. If you need a break, you can come back and log in to your account at fileyourstatetaxes.org.
+
+            Need help? Reply to this email.
+
+            Best,
+            The FileYourStateTaxes team
+
+            If you would like to unsubscribe from emails, click here.
```
Comment (mpidcock) [inline config/locales/en.yml:1549]: TODO: Get the real unsubscribe link and add it here

**Review by mpidcock — APPROVED** [review]
💌

File: config/locales/en.yml [hunk 3]
```diff
@@ -1533,6 +1533,25 @@ en:
...
+          subject: Welcome to FileYourStateTaxes
+        sms: |
+          Hi - We successfully created your FileYourStateTaxes account! We'll send you updates on your state tax return status once you submit it. While you are filing, your responses will be saved as you go. If you need a break, you can come back and log in to your account at fileyourstatetaxes.org.
```
[the remark this answers is not in the document]
Comment (mrotondo) [reply config/locales/en.yml:1552]: I haven't seen it reproduced personally and we're not sending SMS yet, but I'll drop the period just for the meme

**Pull request merged**